.github/workflows/fedora33/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM fedora:33
+
+# install dependencies
+RUN dnf -y install \
+      https://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm \
```

Comment:
Can we add a comment why we're using these (slightly) non-standard sources?

Reply:
Yeah no prob, ill put which required packages are in those repos

Reply:
Ends up a lot of them weren't necessary;

Seems there was some Docker weirdness happening on my end (I've been testing in standalone containers, since pushing to Github all the time to test got... old, quick)

Reply:
I've been using the RPM Fusion repos locally since reading one of the Fedora guides for getting Audio/Video working:

https://docs.fedoraproject.org/en-US/quick-docs/assembly_installing-plugins-for-playing-movies-and-music/

Ends up I didnt need it here (**gstreamer1-plugins-good**, have the *replaygain* plugin that one of the tests uses, and it's in the official Fedora 33 repo)